utils/kata-manager.sh

```diff
@@ -146,7 +146,23 @@ github_get_release_file_url()
 			;;
 
 		*containerd*)
-			[ "$arch" = "x86_64" ] && arch="amd64"
+			case "${arch}" in
+				x86_64*)
+					arch=amd64
+					;;
```

Comment:
The code as-is is very clear, but it might be worth considering collapsing each case into a single line for quicker reading, For example:

```suggestion
				x86_64*) arch=amd64 ;;
```